### ISSUE TITLE
Need different folders for different levels of difficulty.

### ISSUE THREAD

**Issue by Ajeet1606** [QasimWani/LeetHub#111]
As of now all problems are simply pushed in the repo, so the length of repo on the front itself becomes very long. It would be great help if we can get a feature that problems are added in different folders according to their difficulty.
Like most commonly 3 folders, one consisting of easy problems, one medium and one for hard.
Thanks!

**Comment by vikas0913** [comment]
> As of now all problems are simply pushed in the repo, so the length of repo on the front itself becomes very long. It would be great help if we can get a feature that problems are added in different folders according to their difficulty. Like most commonly 3 folders, one consisting of easy problems, one medium and one for hard. Thanks!

To add, just adding a general folder structure support would be beneficial, especially if we want to link leethub to an existing repo (with a specific code/folder structure already in place).

**Comment by Kajal13081** [comment]
> > As of now all problems are simply pushed in the repo, so the length of repo on the front itself becomes very long. It would be great help if we can get a feature that problems are added in different folders according to their difficulty. Like most commonly 3 folders, one consisting of easy problems, one medium and one for hard. Thanks!
> 
> To add, just adding a general folder structure support would be beneficial, especially if we want to link leethub to an existing repo (with a specific code/folder structure already in place).

Yes like if we are solving questions of array, so we can make an array folder where all array questions can be stored. In this way we can have all the questions topic wise.

**Comment by SHIVAM2001-NITP** [comment]
gfg practice code auto push also ?

**Comment by Heisenberg0203** [comment]
Different folders for different topics would be beneficial.

**Comment by NikitaShkaruba** [comment]
It would be nice to have a feature of migrating an existing repo to the new file structure also! :)

**Comment by samipsuwal** [comment]
@QasimWani My latest push https://github.com/QasimWani/LeetHub/pull/230 implements this

**Comment by ZenithClown** [comment]
I can see PR #369 but how to get this done?

**Comment by singh3abhi** [comment]
Hey i just wanted to know if anyone implemented the folder structure support in their forked repos
which will sync problems to their tags -> difficulty -> problem

**Comment by RaghaRao314159** [comment]
I am unable to link contest problem solutions, It would be great if this gets added as well

**Comment by Ajeet1606** [comment]
@RaghaRao314159 as far as I know contest problems aren't pushed automatically, submit the code to online judge after contest, it should work then.

**Comment by Reena-senthilkumar** [comment]
I couldn't able to connect leetcode with github
